Documentation/ceph-cluster-crd.md

```diff
@@ -519,7 +519,12 @@ When other keys are set, `all` will be merged together with the specific compone
 
 ### Placement Configuration Settings
 
-Placement configuration for the cluster services. It includes the following keys: `mgr`, `mon`, `osd`, `cleanup`, and `all`. Each service will have its placement configuration generated by merging the generic configuration under `all` with the most specific one (which will override any attributes).
+Placement configuration for the cluster services. It includes the following keys: `mgr`, `mon`, `arbiter`, `osd`, `cleanup`, and `all`.
+Each service will have its placement configuration generated by merging the generic configuration under `all` with the most specific one (which will override any attributes).
+
+In stretch clusters, if the `arbiter` placement is specified, that placement will only be applied to the arbiter.
+The remaining mons will still use the `mon` and/or `all` sections.
```

Comment:
I would be a little clearer here that `arbiter` doesn't merge with `all`. It's implied, but I can see someone still assuming the merge.